### PR TITLE
Add documentation about singularity installation

### PR DESCRIPTION
**Pull request by jaimesouza**
Add a documentation about the singularity installation in the slurmd charm.

### CODE REVIEW

**Review by heitorPB — CHANGES_REQUESTED** [review]
Thanks! Here's some feedback for you:

**Comment by jaimesouza** [comment]
@heitorPB suggestions commited. Is it approved?

File: new_words_spell_checker.txt [hunk 1]
```diff
@@ -57,3 +57,4 @@ Soltan
 systemd
 Tosta
 Zeestraten
+Heitor
```
Comment (heitorPB) [inline new_words_spell_checker.txt:60]: I wonder why this is needed? Can you keep this file alphabetically ordered please?

Comment (jaimesouza) [reply new_words_spell_checker.txt:60]: Without it, the spell checker fails. I am gonna get it ordered.

**Review by heitorPB — APPROVED** [review]
Nice work!

**Pull request merged**